### PR TITLE
create calculator wireframe

### PR DESCRIPTION
**Pull request by tjay1760**
In this pull request :

- -I deleted all the default react BoilerPlate
- Create a new directory called components.
- Inside components, I created a new Calculator.js file.
- Implemented the Styling using CSS and JSX
- Used functional components
- Created at least 1 child component

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,56 @@
-.App {
```
Comment (amejid) [inline src/App.css:1]: - [ ] You did a great job with the design. :+1: Kindly ensure to remove the boilerplate files created by CRA as suggested in the project requirements. 🥸️

Screenshot from 2023-07-06 13-00-12

- The following files can be considered boilerplate files.

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
```
Comment (amejid) [inline package.json:6]: - [ ] Your README file is returning 404 Not Found when clicking on the MIT license link. Kindly add a License file to make your project more professional. 🤝️

Screenshot from 2023-07-06 12-59-05

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @tjay1760, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Great UI design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @tjay1760, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**